code/modules/clothing/gloves/tacklers.dm

```diff
@@ -39,6 +43,14 @@
 	if(H.get_item_by_slot(ITEM_SLOT_GLOVES) == src)
 		qdel(tackler)
 
+/obj/item/clothing/gloves/tackler/AltClick(mob/user)
+	if(user.canUseTopic(src, BE_CLOSE))
+		..()
+		if(istype(user) && loc == user)
+			var/input = stripped_input(user,"What do you want to yell when you launch a tackle?.", ,"", 50)
```

Comment:
can you make your quip contain a forbidden word (like "reddit") so that you won't say anything when you use the gloves?

Reply:
other battle cries allow forbidden words so i'm following suit unless directed otherwise